packages/cli/generation/ir-generator/src/filtered-ir/FilteredIr.ts

```diff
@@ -140,8 +140,8 @@ export class FilteredIrImpl implements FilteredIr {
 
     public hasWebhookPayloadProperty(webhookId: string, property: string): boolean {
         const properties = this.webhookPayloadProperties[webhookId];
-        // no properties were filtered for inlined request
         if (properties == null) {
+            // No audiences were configured.
```

Comment:
@amckinney how does this check differ from if the value is `undefined` vs. the value is not present in the map?